### PR TITLE
fix: Use podman/hello as the example podman container

### PR DESCRIPTION
**Pull request by rhatdan**
the image redhat/ubi8-micro does not exists.

It should be just ubi8-micro, but I think this image is better.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by benoitf** [comment]
@rhatdan it exists

```
podman run redhat/ubi8-micro
hello world
```

but I'm fine switching to the podman hello world

**Comment by benoitf** [comment]
on my side I've error using podman/hello image

```
$ podman run -ti podman/hello 
Resolving "podman/hello" using unqualified-search registries (/etc/containers/registries.conf.d/999-podman-machine.conf)
Trying to pull docker.io/podman/hello:latest...
Error: initializing source docker://podman/hello:latest: reading manifest latest in docker.io/podman/hello: errors:
denied: requested access to the resource is denied
unauthorized: authentication required
```

**Comment by benoitf** [comment]
so what's the FQN of `podman/hello` image as it's not on `docker.io` ?

**Comment by slemeur** [comment]
Also @rhatdan, shouldn't it be consistent with documentation on Podman?
- https://docs.podman.io/en/latest/Introduction.html 
- https://podman.io/getting-started/#running-a-container

The podman/hello looks good, but shouldn't it also be used on podman's docs?

**Comment by rhatdan** [comment]
https://github.com/containers/shortnames/pull/40

Added hello to podman shortnames.

**Comment by rhatdan** [comment]
We should not base all of our images off of docker.io.  Maybe we should fully address this for now as quay.io/podman/hello

**Comment by rhatdan** [comment]
quay.io/podman/hello

File: packages/renderer/src/lib/container/ContainerEmptyScreen.svelte [hunk 1]
```diff
@@ -24,7 +24,7 @@ onMount(async () => {
       <h1 class="pf-c-title pf-m-lg">No containers</h1>
       <div class="pf-c-empty-state__body">Run a first container using the following command line:</div>
       <div class="flex flex-row bg-gray-800 w-full items-center p-2 mt-2">
-        <div id="noContainerCommandLine">podman run redhat/ubi8-micro echo hello world</div>
+        <div id="noContainerCommandLine">podman run -ti quay.io/podman/hello</div>
```
Comment (benoitf) [inline packages/renderer/src/lib/container/ContainerEmptyScreen.svelte]: We don't need the -ti flag to echo the output so I would keep it simple

```suggestion
        <div id="noContainerCommandLine">podman run quay.io/podman/hello</div>
```

File: website/docs/intro.md [hunk 2]
```diff
@@ -20,5 +20,21 @@ Get started by **[Downloading](/downloads) the product** for your Operating Syst
 Run the development server:
 
 ```sh
-$ podman run redhat/ubi8-micro echo hello world
+$ podman run -ti quay.io/podman/hello
```
Comment (benoitf) [inline website/docs/intro.md]:
```suggestion
$ podman run quay.io/podman/hello
```

**Comment by benoitf** [comment]
> We should not base all of our images off of docker.io. Maybe we should fully address this for now as quay.io/podman/hello

sure. I was looking for such hello image from podman but it seems it was well hidden as getting started of podman.io is using `docker.io` as a sample https://podman.io/getting-started/#running-a-container

**Comment by rhatdan** [comment]
Yes this is a difference between a Mac and Linux.  On a Mac we can not prompt so we go directly to docker.io (I hate that.)
On Linux it prompts and remembers.

I have an update to containers-common in the works which will allow us to use podman run hello, and make it much nicer. Without having to mention docker.io.

**Review by benoitf — APPROVED** [review]
thanks: much better with an official hello world podman image 👍

**Comment by benoitf** [comment]
@rhatdan feel free to merge it

**Pull request merged**